academy/3-my-own-chain/create-message.md

```diff
@@ -65,51 +65,66 @@ Currently:
 
 Now you need a message to instruct the checkers blockchain to create a game. This message needs to:
 
-* Not specify the creator: this is implicit because it shall be the signer of the message.
-* Not specify the ID of the game, because the system uses an incrementing counter. However, the server needs to return the newly created ID value, since the eventual value cannot be known before the transaction is included in a block and the state computed. Call this `idValue`.
+* Not specify the ID of the game, because the system uses an incrementing counter. However, the server needs to return the newly created ID value, since the eventual value cannot be known before the transaction is included in a block and the state computed. Call this `gameIndex`.
 * Not specify the game board as this is controlled by the checkers rules.
-* Specify who is playing with the red pieces. Call the field `red`.
 * Specify who is playing with the black pieces. Call the field `black`.
+* Specify who is playing with the red pieces. Call the field `red`.
 
 Instruct Ignite CLI to do all of this:
 
+<CodeGroup>
+
+<CodeGroupItem title="Local" active>
+
+```sh
+$ ignite scaffold message createGame black red --module checkers --response gameIndex
+```
+
+</CodeGroupItem>
+
+<CodeGroupItem title="Docker">
+
 ```sh
-$ ignite scaffold message createGame red black --module checkers --response idValue
+$ docker run --rm -it -v $(pwd):/checkers -w /checkers checkers_i ignite scaffold message createGame black red --module checkers --response gameIndex
 ```
 
-This creates a [certain number of files](https://github.com/cosmos/b9-checkers-academy-draft/commit/e78cba34926ba0adee23febb1ce44774e2c466b3) plus [some GUI elements](https://github.com/cosmos/b9-checkers-academy-draft/commit/dcf9f4724570146c8e0ad339aafb469d27dca0b9).
+</CodeGroupItem>
+
+</CodeGroup>
+
+This creates a [certain number of files](https://github.com/cosmos/b9-checkers-academy-draft/commit/66ae6e1) plus [some GUI elements](https://github.com/cosmos/b9-checkers-academy-draft/commit/0365f55).
 
 ## Protobuf objects
 
 Simple Protobuf objects are created:
 
-```protobuf [https://github.com/cosmos/b9-checkers-academy-draft/blob/eea2618/proto/checkers/tx.proto#L15-L23]
+```protobuf [https://github.com/cosmos/b9-checkers-academy-draft/blob/create-game-msg/proto/checkers/tx.proto#L14-L22]
 message MsgCreateGame {
     string creator = 1;
-    string red = 2;
-    string black = 3;
+    string black = 2;
```

Comment:
Just curious why we needed a proto breaking change between v1 and v2

Reply:
Yoo! Sorry I don't know what happened, this PR should have a single file to review: `migration.md`